node_modules/gh-rest/lib/rest/user.js

```diff
@@ -33,6 +33,29 @@ module.exports = function(client) {
     };
 
     /**
+     * Get all users for an app
+     *
+     * @param  {Number}         [app]                           The application to search users in, defaults to the current application
+     * @param  {String}         [filter]                        The string to filter users by
```

Comment:
Should we call this `query`. `filter` suggest that something more advanced is going to be possible